plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabProjectAccessTokenAction.examples.test.ts

```diff
@@ -13,23 +13,25 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import yaml from 'yaml';
-import { createGitlabProjectAccessTokenAction } from './createGitlabProjectAccessTokenAction'; // Adjust the import based on your project structure
-import { ScmIntegrations } from '@backstage/integration';
-import { ConfigReader } from '@backstage/config';
 import { getVoidLogger } from '@backstage/backend-common';
+import { ConfigReader } from '@backstage/config';
+import { ScmIntegrations } from '@backstage/integration';
 import { PassThrough } from 'stream';
+import yaml from 'yaml';
+import { createGitlabProjectAccessTokenAction } from './createGitlabProjectAccessTokenAction'; // Adjust the import based on your project structure
```

Comment:
Don't think you need this comment.

```suggestion
import { createGitlabProjectAccessTokenAction } from './createGitlabProjectAccessTokenAction';
```